deploy/createTypesPackages.mjs

```diff
@@ -0,0 +1,122 @@
+// @ts-check
+
+// node deploy/createTypesPackages.mjs
+
+// prettier-ignore
+const packages = [
+    {
+      name: "@types/web",
+      description: "Types for the DOM, and other web technologies in browsers",
+      files: [
+        { from: "../generated/dom.generated.d.ts", to: "index.d.ts" }
+      ],
+    },
+  ];
+
+// Note: You can add 'version: "1.0.0"' to a package above
+// to set the major or minor, otherwise it will always bump
+// the patch.
+
+import { join, dirname } from "path";
+import fs from "fs";
+import { fileURLToPath } from "url";
+import pkg from "prettier";
+const { format } = pkg;
+import { execSync } from "child_process";
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const go = async () => {
+  const gitSha = execSync("git rev-parse HEAD").toString().trim().slice(0, 7);
+
+  const generatedDir = join(__dirname, "generated");
+  const templateDir = join(__dirname, "template");
+
+  for (const pkg of packages) {
+    const folderName = pkg.name.replace("@", "").replace("/", "-");
+    const packagePath = join(generatedDir, folderName);
+
+    if (fs.existsSync(packagePath)) fs.rmSync(packagePath, { recursive: true });
+    fs.mkdirSync(packagePath, { recursive: true });
+
+    // Migrate in the template files
+    for (const templateFile of fs.readdirSync(templateDir)) {
+      if (templateFile.startsWith(".")) continue;
```

Comment:
Is this line used? Is it for .DS_Store?

Reply:
Yep, on macOS they appear all over the place and I just always do this